Don't pin dependencies because MacPorts already has newer versions than
these.
https://jira.mongodb.org/browse/SERVER-62686
Reverts:
https://github.com/mongodb/mongo/commit/76fb7482945275ef14dbad296b4fd89dbf5eb385
https://github.com/mongodb/mongo/commit/9ec376a973af7e54e21fc15b4f63579304891d8f
--- etc/pip/components/aws.req.orig	2022-01-21 17:22:14.000000000 -0600
+++ etc/pip/components/aws.req	2022-01-28 16:48:40.000000000 -0600
@@ -1,2 +1,3 @@
-boto3 <= 1.20.26
-botocore <= 1.23.26
+boto3
+botocore
+psutil
--- etc/pip/components/compile.req.orig	2022-01-21 17:22:14.000000000 -0600
+++ etc/pip/components/compile.req	2022-01-28 16:48:40.000000000 -0600
@@ -1,5 +1,6 @@
 # Mongo compile
-Cheetah3 <= 3.2.6.post1 # src/mongo/base/generate_error_codes.py
-regex <= 2021.11.10
-requirements_parser <= 0.3.1
+Cheetah3 # src/mongo/base/generate_error_codes.py
+psutil
+regex
+requirements_parser
 setuptools
--- etc/pip/components/core.req.orig	2022-01-21 17:22:14.000000000 -0600
+++ etc/pip/components/core.req	2022-01-28 16:48:40.000000000 -0600
@@ -1,8 +1,8 @@
 # Core (we need these for most buildscripts)
-psutil <= 5.8.0
-pymongo >= 3.9, < 4.0
-PyYAML >= 3.0.0, <= 6.0.0
-requests >= 2.0.0, <= 2.26.0
+psutil
+pymongo >= 3.9, != 3.6.0  # See PYTHON-1434, SERVER-34820
+PyYAML >= 3.0.0
+requests >= 2.0.0
 pkce == 1.0.3
 oauthlib == 3.1.1
 requests-oauthlib == 1.3.0
\ No newline at end of file
--- etc/pip/components/idl.req.orig	2022-01-21 17:22:14.000000000 -0600
+++ etc/pip/components/idl.req	2022-01-28 16:48:17.000000000 -0600
@@ -1,3 +1,3 @@
-unittest-xml-reporting >= 2.2.0, <= 3.0.4
-typing <= 3.7.4.3
-packaging <= 21.3
+unittest-xml-reporting>=2.2.0
+typing
+packaging
--- etc/pip/components/jiraclient.req.orig	2022-01-21 17:22:14.000000000 -0600
+++ etc/pip/components/jiraclient.req	2022-01-28 16:48:17.000000000 -0600
@@ -1,6 +1,6 @@
 # TIG jira integration
-jira <= 3.1.1
-requests-oauth <= 0.4.1
-PyJWT <= 2.3.0 # https://github.com/pycontribs/jira/issues/247
+jira
+requests-oauth
+PyJWT # https://github.com/pycontribs/jira/issues/247
 cryptography == 2.3 # Needed for oauthlib to use RSAAlgorithm # Version locked - see SERVER-36618
 # We are omitting pycrypto based on https://github.com/pycontribs/jira/pull/629
--- etc/pip/components/lint.req.orig	2022-01-21 17:22:14.000000000 -0600
+++ etc/pip/components/lint.req	2022-01-28 16:48:17.000000000 -0600
@@ -5,7 +5,7 @@
 pydocstyle == 2.1.1
 pylint == 2.7.2
 structlog ~= 19.2.0
-typing <= 3.7.4.3
+typing
 yamllint == 1.15.0
 yapf == 0.26.0
 evergreen-lint == 0.1.3
--- etc/pip/components/resmoke.req.orig	2022-01-21 17:22:14.000000000 -0600
+++ etc/pip/components/resmoke.req	2022-01-28 16:48:17.000000000 -0600
@@ -1,9 +1,9 @@
 curatorbin == 1.2.1
 PyKMIP == 0.4.0 # It's now 0.8.0. We're far enough back to have API conflicts.
 evergreen.py == 3.1.0
-jinja2 <= 2.11.3
+jinja2
 MarkupSafe == 1.1.0  # See SERVER-57036, this is a transitive dependency of jinja2
-mock <= 4.0.3
+mock
 shrub.py == 1.1.4
 ocspresponder == 0.5.0
 flask == 1.1.1
@@ -13,5 +13,5 @@
 googleapis-common-protos == 1.53.0
 blackduck == 1.0.1
 PyGithub == 1.53
-urllib3 >= 1.26.0, <= 1.26.7
+urllib3 >= 1.26.0
 distro == 1.5.0
